llvm/include/llvm/Bitcode/LLVMBitCodes.h

```diff
@@ -681,12 +681,14 @@ enum FunctionCodes {
       61, // [DILocation, DILocalVariable, DIExpression, ValueAsMetadata]
   FUNC_CODE_DEBUG_RECORD_DECLARE =
       62, // [DILocation, DILocalVariable, DIExpression, ValueAsMetadata]
+  FUNC_CODE_DEBUG_RECORD_COROFRAME_ENTRY =
+      64, // [DILocation, DILocalVariable, DIExpression, ValueAsMetadata]
   FUNC_CODE_DEBUG_RECORD_ASSIGN =
       63, // [DILocation, DILocalVariable, DIExpression, ValueAsMetadata,
           //  DIAssignID, DIExpression (addr), ValueAsMetadata (addr)]
   FUNC_CODE_DEBUG_RECORD_VALUE_SIMPLE =
-      64, // [DILocation, DILocalVariable, DIExpression, Value]
-  FUNC_CODE_DEBUG_RECORD_LABEL = 65, // [DILocation, DILabel]
+      65, // [DILocation, DILocalVariable, DIExpression, Value]
+  FUNC_CODE_DEBUG_RECORD_LABEL = 66, // [DILocation, DILabel]
```

Comment:
Let's try to avoid renumbering them, this causes an unnecessary binary incompatibility in the bitcode format.